buying-guide.html

```diff
@@ -8,78 +8,129 @@
-			</header>
-			{{ question_set.question.helper_text }}
+<div class="buying-guide">
+  {% for question_set in site.data.buying_guide %}
+  	<div class="row question {{ question_set.question.initial_state }}" id="question-{{ question_set.id }}" data-question-index="{{ question_set.id }}">
+  		<section class="8u -2u">
+  			<header>
+  				<h2 class="question">{{ question_set.question.text }}</a></h2>
+  			</header>
 
-			<p class="responses">
-				{% for response_option in question_set.responses %}
-					<div class="response">
-            {% if response_option.image_path %}
-              <img src="{{ response_option.image_path }}" width="300px" />
-            {% endif %}
-						<a href="#" data-question-result="{{ response_option.result }}">{{ response_option.text }}</a>
-					</div>
-				{% endfor %}
-			</p>
-		</section>
-	</div>
-{% endfor %}
+  			{{ question_set.question.helper_text }}
 
-<div class="row result hidden" id="question-success">
-	<section class="8u -2u">
-		<header>
-			<h2>Home is Compatible</a></h2>
-		</header>
+  			<p class="responses">
+          <div class="row">
+            {% for response_option in question_set.responses %}
+              {% assign grid_width = 12 | divided_by: question_set.responses.size %}
 
-		<p>
-			Yay! Your home is compatible with Glow. You'll be saving money and helping the environment before you know it. <a href="{{ site.share.kickstarter }}">Support us on kickstarter</a> now!
-		</p>
-	</section>
-</div>
+              {% if response_option.image_path %}
+    					  <div class="{{ grid_width }}u 12u$(small) response">
+                  <a href="#" class="image fit" data-question-result="{{ response_option.result }}">
+                    <img src="{{ response_option.image_path }}" class="fit" /><br />
+                    {{ response_option.text }}
+                  </a>
+                </div>
+              {% else %}
+                <div class="{{ grid_width }}u response">
+                  <a href="#" class="button" data-question-result="{{ response_option.result }}">
+                    {{ response_option.text }}
+                  </a>
+                </div>
+              {% endif %}
+    				{% endfor %}
+          </div>
+  			</p>
+  		</section>
+  	</div>
+  {% endfor %}
 
-<div class="row result hidden" id="question-failure">
-	<section class="8u -2u">
-		<header>
-			<h2>Home is not Compatible</a></h2>
-		</header>
+  <div class="row result hidden" id="question-success">
+  	<section class="8u -2u">
+  		<header>
+  			<h2>Glow is compatible!</a></h2>
+  		</header>
 
-		<p>
-			We're sorry, but your home is not compatible. Please signup below for details about when we expand abilities to support your house.
-		</p>
-	</section>
-</div>
+  		<p>
+  			Yay! Glow is compatabile with Glow. You'll be saving money and helping the environment before you know it. <br /><a class="button special fit cta" href="{{ site.share.kickstarter }}">Buy Now</a>
```

Comment:
Ha! Good catch. I started with the idea of Home being compatible with a Glow, but then switched it and must have missed it. Clearly, what I wrote doesn't make sense, but how should we think about it?